### PR TITLE
Documentation for scheduling in the very-near-future

### PR DESCRIPTION
**Pull request by tom-con**
#780  

- Adds section to Reliable Scheduling doc about very-near-future scheduled jobs
- Fixes broken link in Troubleshooting to route to appropriate Requirements header

Please feel free to improve any questionable wording! Thanks.

### CODE REVIEW

File: guides/recipes/reliable-scheduling.md [hunk 1]
```diff
@@ -60,3 +60,19 @@ _An [extensive discussion][oi27] on the Oban issue tracker prompted this example
...
+# 1 second from now
+new(schedule_in: 1)
+|> Oban.insert()
+# 500 milliseconds from now
+very_soon = DateTime.utc_now() |> DateTime.add(500, :millisecond)
+new(scheduled_at: very_soon) |> Oban.insert()
+```
+your workers may not be aware of/attempt to perform the job until the next tick as specific by the [Oban Stager][stag] `:interval` option. By default this is set to `1_000ms`.
+
+[stag]: Oban.Plugins.Stager.html
```
Comment (sorentwo) [inline guides/recipes/reliable-scheduling.md:76]: Note to self: update these docs when Stager is no longer a plugin.

**Review by sorentwo — CHANGES_REQUESTED** [review]
The doc changes all look great! However, I would like some formatting changes:

1. Please add space between the header and paragraph
2. Please add space between the paragraph and code block
3. Please hard wrap text blocks to match the rest of the guide

File: guides/recipes/reliable-scheduling.md [hunk 2]
```diff
@@ -60,3 +60,19 @@ _An [extensive discussion][oi27] on the Oban issue tracker prompted this example
...
+
+## Considerations for Scheduling Jobs in the Very-Near-Future
+If you use the `schedule_in` or `scheduled_at` options with a value that resolves to the very-near-future, for example:
+```elixir
+# 1 second from now
+new(schedule_in: 1)
+|> Oban.insert()
+# 500 milliseconds from now
+very_soon = DateTime.utc_now() |> DateTime.add(500, :millisecond)
+new(scheduled_at: very_soon) |> Oban.insert()
```
[the remark this answers is not in the document]
Comment (tom-con) [reply guides/recipes/reliable-scheduling.md]: Thank you for the quick review, requested changes pushed.

**Review by sorentwo — APPROVED** [review]
🎉

**Pull request merged**